### PR TITLE
Make some updates to the contributing sections of the docs

### PR DESCRIPTION
**Pull request by cicdw**
This PR:
- touches up the Contributing section of our docs
- changes the default to light mode (I don't think a documentation site should ever default to dark mode)

### CODE REVIEW

File: docs/contribute/dev-contribute.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Contribute
+title: How to Contribute
 ---
 
 Thanks for contributing to Prefect!
```
Comment (billpalombi) [inline docs/contribute/dev-contribute.mdx:5]:
```suggestion
We welcome your contributions! 
Set up Prefect for development to experiment with extensions and improvements.
```

File: docs/contribute/dev-contribute.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Contribute
+title: How to Contribute
 ---
```
Comment (billpalombi) [inline docs/contribute/dev-contribute.mdx:3]:
```suggestion
description: Learn how to set up Prefect for development.
---
```

File: docs/contribute/dev-contribute.mdx (hunk 1 above)
Comment (billpalombi) [reply docs/contribute/dev-contribute.mdx:5]: Open to other things here but I think we can do better than what was there.

**Review by billpalombi — APPROVED** [review]
I made a couple of "yes, and" recommendations. I'm not wedded to the specific language I proposed.

This change may break some links, but we can handle that separately.

File: docs/contribute/dev-contribute.mdx (hunk 1 above)
Comment (cicdw) [reply docs/contribute/dev-contribute.mdx:6]: @billpalombi i incorporated your change - I don't think it's the right use of my time right now vs. transactions but I want to revisit this page and have a top level section that is explicitly about making Pull Requests / opening Issues, as I think those fall under "Contributing", and I think that will fill out this page nicely.

**Pull request merged**